packages/theme-patternfly-org/components/footer/footer.js

```diff
@@ -27,99 +23,105 @@ export const Footer = () => (
           className="pf-u-mb-lg pf-u-mb-0-on-sm"
         >
           <Grid className="pf-u-py-xl">
-            <GridItem sm={6} md={4} className="pf-u-ml-md pf-u-ml-0-on-md">
+            <GridItem sm={6} md={4} className="pf-u-ml-md pf-u-ml-0-on-md pf-u-ml-md pf-u-mb-xl pf-u-mb-0-on-md">
```

Comment:
Duplicate `pf-u-ml-md` className
```suggestion
            <GridItem sm={6} md={4} className="pf-u-ml-md pf-u-ml-0-on-md  pf-u-mb-xl pf-u-mb-0-on-md">
```